fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/DeleteFromUsingCommand.java

```diff
@@ -77,8 +77,8 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
                     + ctx.getSessionVariable().printDebugModeVariables());
         }
         // NOTE: delete from using command is executed as insert command, so txn insert can support it
-        new InsertIntoTableCommand(completeQueryPlan(ctx, logicalQuery), Optional.empty(), Optional.empty()).run(ctx,
-                executor);
+        new InsertIntoTableCommand(completeQueryPlan(ctx, logicalQuery), Optional.empty(), Optional.empty(), cte).run(
+                ctx, executor);
```

Comment:
`compeleteQueryPlan` aready process cte, so we should
either use `Optional.empty()` here,
or do not process cte in `compeleteQueryPlan`